python-pyproject-toml/snakesay-project/snakesay/snake.py

```diff
@@ -0,0 +1,22 @@
+SNAKE = r"""  \
+   \    ___
+    \  (o o)
+        \_/ \
+         λ \ \
+           _\ \_
+          (_____)_
+         (________)=Oo°
+"""
+
+
+def bubble(message):
+    bubble_length = len(message) + 2
+    return f"""
+ {"_" * bubble_length}
+( {message} )    
+ {"‾" * bubble_length}"""
+
+
+def say(message):
+    print(bubble(message.replace("s", "ss").replace("S", "SS")))
```

Comment:
I think this is one of the killer features of the new version of `snakesay` that makes it stand out above the rest! :star2: